src/create_grid_polar.cpp

```diff
@@ -243,7 +243,9 @@ void level::build_theta()
             theta[i] = fac * i;
     }
     else {
-        std::cout << "Anisotropy chosen in theta.\n";
+        if (gyro::icntl[Param::verbose] > 2) {
+            std::cout << "Anisotropy chosen in theta.\n";
+        }
         ntheta     = pow(2, ceil(log2(nr)) - 1);
```

Comment:
ntheta only depends on nr and not on ntheta_exp.

Further, when we calculate "delete circles" in level::define_line_splitting() (smoother.cpp line 73), we use that the grid spacing in theta is constant, dont we ? If anisotrophy is chosen in theta I think one should update that function as well!

Reply:
https://github.com/mknaranja/GMGPolar/issues/42